src/System.CommandLine.Subsystems.Tests/ErrorReportingSubsystemTests.cs

```diff
@@ -0,0 +1,84 @@
+
+namespace System.CommandLine.Subsystems.Tests;
+
+public class ErrorReportingSubsystemTests
+{
+    [Fact]
+    public void Report_when_single_error_writes_to_console_hack()
+    {
+        var error = new ParseError("a sweet error message");
+        var errors = new List<ParseError> { error };
+        var errorSubsystem = new ErrorReportingSubsystem();
+        var consoleHack = new ConsoleHack().RedirectToBuffer(true);
+
+        errorSubsystem.Report(consoleHack, errors);
+
+        consoleHack.GetBuffer().Trim().Should().Be(error.Message);
+    }
+
+    [Fact]
+    public void Report_when_multiple_error_writes_to_console_hack()
+    {
+        var error = new ParseError("a sweet error message");
+        var anotherError = new ParseError("another sweet error message");
+        var errors = new List<ParseError> { error, anotherError };
+        var errorSubsystem = new ErrorReportingSubsystem();
+        var consoleHack = new ConsoleHack().RedirectToBuffer(true);
+
+        errorSubsystem.Report(consoleHack, errors);
+
+        consoleHack.GetBuffer().Trim().Should().Be($"{error.Message}{Environment.NewLine}{anotherError.Message}");
+    }
+
+    [Fact]
+    public void Report_when_no_errors_writes_nothing_to_console_hack()
+    {
+        var errors = new List<ParseError> { };
+        var errorSubsystem = new ErrorReportingSubsystem();
+        var consoleHack = new ConsoleHack().RedirectToBuffer(true);
+
+        errorSubsystem.Report(consoleHack, errors);
+
+        consoleHack.GetBuffer().Trim().Should().Be("");
+    }
+
+    [Theory]
+    [InlineData("-x", true)]
+    [InlineData("-non_existant_option", true)]
+    public void GetIsActivated_GivenInvalidInput_ShouldReturnTrue(string input, bool result)
+    {
+        var rootCommand = new CliRootCommand {new CliOption<bool>("-v")};
+        var configuration = new CliConfiguration(rootCommand);
+        var errorSubsystem = new ErrorReportingSubsystem();
+        IReadOnlyList<string> args = [""];
+        Subsystem.Initialize(errorSubsystem, configuration, args);
+
+        var parseResult = CliParser.Parse(rootCommand, input, configuration);
+        var isActive = Subsystem.GetIsActivated(errorSubsystem, parseResult);
+
+        isActive.Should().Be(result);
+    }
+
+    [Theory]
+    [InlineData("-v", false)]
+    [InlineData("", false)]
+    public void GetIsActivated_GivenValidInput_ShouldReturnFalse(string input, bool result)
+    {
+        var rootCommand = new CliRootCommand { new CliOption<bool>("-v") };
+        var configuration = new CliConfiguration(rootCommand);
+        var errorSubsystem = new ErrorReportingSubsystem();
+        IReadOnlyList<string> args = [""];
+        Subsystem.Initialize(errorSubsystem, configuration, args);
+
+        var parseResult = CliParser.Parse(rootCommand, input, configuration);
+        var isActive = Subsystem.GetIsActivated(errorSubsystem, parseResult);
+
+        isActive.Should().Be(result);
+    }
```

Comment:
```suggestion
    [Theory]
    [InlineData("-v")]
    [InlineData("")]
    public void GetIsActivated_GivenValidInput_SubsystemShouldBeActive(string input)
    {
        var rootCommand = new CliRootCommand { new CliOption<bool>("-v") };
        var configuration = new CliConfiguration(rootCommand);
        var errorSubsystem = new ErrorReportingSubsystem();
        IReadOnlyList<string> args = [""];
        Subsystem.Initialize(errorSubsystem, configuration, args);

        var parseResult = CliParser.Parse(rootCommand, input, configuration);
        var isActive = Subsystem.GetIsActivated(errorSubsystem, parseResult);

        isActive.Should().BeFalse();
    }
```
No need for the second result parameter

Is this test name slightly more clear?

Reply:
Thanks for catching that second arg, and I think those test names are even clearer. Really drives home what the boolean values for GetIsActivated actually mean in business logic 👍 

Appreciate your feedback